fluss-server/src/main/java/com/alibaba/fluss/server/kv/rocksdb/RocksDBMetricsCollector.java

```diff
@@ -0,0 +1,527 @@
+                MetricNames.ROCKSDB_NUM_FILES_AT_LEVEL_5, (Gauge<Long>) () -> numFilesAtLevel5);
+        bucketMetricGroup.gauge(
+                MetricNames.ROCKSDB_NUM_FILES_AT_LEVEL_6, (Gauge<Long>) () -> numFilesAtLevel6);
+
+        bucketMetricGroup.gauge(
+                MetricNames.ROCKSDB_WRITE_STALL_MICROS, (Gauge<Long>) () -> stallTimeMicros);
+
+        // Compression and compaction pressure metrics
+        bucketMetricGroup.gauge(
+                MetricNames.ROCKSDB_COMPACTION_PENDING, (Gauge<Long>) () -> compactionPending);
+
+        bucketMetricGroup.gauge(
+                MetricNames.ROCKSDB_FLUSH_PENDING, (Gauge<Long>) () -> flushPending);
+        bucketMetricGroup.gauge(
+                MetricNames.ROCKSDB_NUM_RUNNING_COMPACTIONS,
+                (Gauge<Long>) () -> numRunningCompactions);
+        bucketMetricGroup.gauge(
+                MetricNames.ROCKSDB_NUM_RUNNING_FLUSHES, (Gauge<Long>) () -> numRunningFlushes);
+
+        // Background job metrics
+        bucketMetricGroup.gauge(
+                MetricNames.ROCKSDB_BACKGROUND_ERRORS, (Gauge<Long>) () -> backgroundErrors);
+
+        // Histogram metrics
+        bucketMetricGroup.gauge(
+                MetricNames.ROCKSDB_DB_GET_LATENCY_MICROS, (Gauge<Long>) () -> dbGetLatencyMicros);
+        bucketMetricGroup.gauge(
+                MetricNames.ROCKSDB_DB_WRITE_LATENCY_MICROS,
+                (Gauge<Long>) () -> dbWriteLatencyMicros);
+        bucketMetricGroup.gauge(
+                MetricNames.ROCKSDB_COMPACTION_TIME_MICROS,
+                (Gauge<Long>) () -> compactionTimeMicros);
+        bucketMetricGroup.gauge(
+                MetricNames.ROCKSDB_COMPRESSION_TIME_NANOS,
+                (Gauge<Long>) () -> compressionTimeNanos);
+        bucketMetricGroup.gauge(
+                MetricNames.ROCKSDB_DECOMPRESSION_TIME_NANOS,
+                (Gauge<Long>) () -> decompressionTimeNanos);
+    }
+
+    public void updateMetrics() {
+        try {
+            // Update cache metrics
+            blockCacheHitCount = getTickerValue(TickerType.BLOCK_CACHE_HIT);
+            blockCacheMissCount = getTickerValue(TickerType.BLOCK_CACHE_MISS);
+            indexBlockCacheHitCount = getTickerValue(TickerType.BLOCK_CACHE_INDEX_HIT);
+            indexBlockCacheMissCount = getTickerValue(TickerType.BLOCK_CACHE_INDEX_MISS);
+            filterBlockCacheHitCount = getTickerValue(TickerType.BLOCK_CACHE_FILTER_HIT);
+            filterBlockCacheMissCount = getTickerValue(TickerType.BLOCK_CACHE_FILTER_MISS);
+            dataBlockCacheHitCount = getTickerValue(TickerType.BLOCK_CACHE_DATA_HIT);
+            dataBlockCacheMissCount = getTickerValue(TickerType.BLOCK_CACHE_DATA_MISS);
+
+            // Update compaction metrics
+            compactionCount =
+                    getTickerValue(TickerType.COMPACT_READ_BYTES) > 0
+                            ? 1
+                            : 0; // Simplified since exact count may not be available
```

Comment:
It's confusing to report a count that only has values 0 and 1. Is it necessary to report this metric? Can `compactionBytesRead` be satisfied?